documentation/modules/auxiliary/scanner/http/http_put.md

```diff
@@ -15,30 +15,26 @@ This module can abuse misconfigured web servers to upload and delete web content
 
 ### ACTION
 
+Set `ACTION` to either `PUT` or `DELETE`. (Default: `PUT`)
+
 **PUT**
 
 Action is set to PUT to upload files to the server. If `FILENAME` isn't specified, the module will generate a random string as a .txt file.
 
 **DELETE** 
 
-Deletes the file specified in the `FILENAME` option. (default: `msf_http_put_test.txt`)
+`FILENAME` is required when Action is set to DELETE 
```

Comment:
i think this line and the line that was removed are both relevant here.  The default file name is still good, and the explanation was solid.